src/retriever/data_tiers/tier_0/dgraph/transpiler.py

```diff
@@ -413,6 +425,44 @@ def _convert_constraints_to_filters(
 
         return filters
 
+    def _convert_qualifier_constraints_to_filter(
+        self,
+        qualifier_constraints: Sequence[QualifierConstraintDict],
+    ) -> str:
+        """Convert TRAPI qualifier_constraints into a Dgraph filter string.
+
+        Within a single `qualifier_set`, items are ORed. Multiple sets are ANDed.
```

Comment:
You've got this backwards. Within one set, items are ANDed, and between sets, they are ORed